README.md

```diff
@@ -23,18 +23,12 @@ You can see a Flask app using the comment scraper on Glitch here: https://glitch
 You must configure a developer token for authentication. You can do that using the instructions in the [Create Developer Tokens](https://docs.frame.io/docs/authentication#section-create-developer-tokens) section of Frame.io's developer docs. 
 
 # Setup
-The sample is written in Python 3. You run it by doing the following:
+The code can be run in Python 2.7 or 3.x. 
 
-1. Open **comment_scrape.py**.
+Before running the code, source TOKEN (your authenticated API token) and ROOT_ASSET_ID, which is the root ID for the project to scrape comments from.  
 
-2. Put the root asset ID you want to retrieve info for in for the ROOT_ASSET_ID value at the top. 
-
-3. Put your developer token in for TOKEN at the top. 
-
-# Usage
-
-1. Run the sample with `$ python comment_scrape.py`
-
-2. Retrieve the .csv file. It will appear in the same folder that you run comment_scrape.py from and be named **output.csv**.
+# Run the code
 
+1. `$ python -i comment_scraper.py`
```

Comment:
it might be a more useful api if, instead of outputting to a file, you defaulted to STDOUT. then someone could pipe it into a file themselves or do something fancy like parsing it with shell commands. or allow something like `-o output.csv`.